api/src/main/java/org/apache/iceberg/types/Conversions.java

```diff
@@ -37,7 +37,7 @@ public class Conversions {
 
   private Conversions() {}
 
-  private static final String HIVE_NULL = "__HIVE_DEFAULT_PARTITION__";
+  public static final String HIVE_NULL = "__HIVE_DEFAULT_PARTITION__";
```

Comment:
I'd rather not make this public and expose hive concepts for narrow use cases through the API, I know it's a bit duplicative but could we just have a private static constant in `Spark3Util` as well (especially considering at some point we can move away from relying on this in the SparkPartition code, and build actual `PartitionData` instances)

Reply:
updated